src/charts/createTaskChart.js

```diff
@@ -0,0 +1,55 @@
+const d3 = require('d3');
+
+/**
+Percentage is a decimal integer.  
+
+Function creates a donut chart based on percentage and color to differentiate
+between Complete, In Progress, and Todo.
+**/
+
+export function createTaskChart(percentage, color) {
```

Comment:
Ah yeah I used my own React D3 Library to create the charts.  The charts itself is written in D3, and yes I do believe if the team is not using D3 then another easier library would be a better choice to use.  I wanted to match the donut chart as close as possible, and the fastest route for that was using D3 for its flexibility from my end.